plugin/init.lua

```diff
@@ -68,6 +64,11 @@ local function apply_to_config(config_builder, plugin_config)
   if plugin_config then
     _smart_splits_wezterm_config.direction_keys = plugin_config.direction_keys
       or _smart_splits_wezterm_config.direction_keys
+    -- update the direction_keys mappings to reflect configured ones
+    for i, v in ipairs(directions) do
+      local key = plugin_config.direction_keys[i]
+      direction_keys[key] = v
+    end
```

Comment:
This will break if you _don't_ pass any config because in that case the `direction_keys` table will just be `{}`.

We need the default to remain in place if you don't pass any config. We can do that by overwriting the table _only if you've passed config._

```suggestion
    -- update the direction_keys mappings to reflect configured ones
    direction_keys = {} -- reset the direction_keys table to overwrite defaults
    for i, v in ipairs(directions) do
      local key = plugin_config.direction_keys[i]
      direction_keys[key] = v
    end
```

Reply:
Gotcha, that's true! I've gone ahead and adjusted the code!